manifests/cf-manifest/spec/manifest/base_properties_spec.rb

```diff
@@ -136,6 +137,13 @@
         )
       }
 
+      it {
+        clients.each { |_, config|
+          expect(config.has_key?("override")).to be true
```

Comment:
Because of rspec magic that translates `have_xxx(foo)` to `has_xxx?(foo)`, you could also say `expect(config).to have_key("override")`

Reply:
That reads more gooder. Thanks.